reactor-core/src/main/java/reactor/core/publisher/FluxSwitchOnFirst.java

```diff
@@ -251,6 +251,25 @@ public Object scanUnsafe(Attr key) {
 			}
 		}
 	}
+	/**
+	 * Adds flags which indicate that the inbound has cancelled upstream and errored
+	 * the outbound downstream. Fails if either inbound is cancelled or terminated.
```

Comment:
again, inconsistency between the doc and the actual code: "either inbound is cancelled or terminated" but code checks for `hasInboundCancelled || hasOutboundCancelled`

Reply:
fixed